server/src/main/java/org/apache/iotdb/db/metadata/MTree.java

```diff
@@ -671,11 +668,11 @@ private int getCountInGivenLevel(MNode node, int targetLevel) {
    *
    * <p>result: [name, alias, storage group, dataType, encoding, compression, offset]
    */
-  List<String[]> getAllMeasurementSchema(ShowTimeSeriesPlan plan) throws MetadataException {
-    List<String[]> res;
-    String[] nodes = MetaUtils.getNodeNames(plan.getPath().getFullPath());
+  List<Pair<PartialPath, String[]>> getAllMeasurementSchema(ShowTimeSeriesPlan plan) throws MetadataException {
```

Comment:
recover this, leave this to another PR， The pair structure is not good also

Reply:
OK